source/backend/functions/settings/src/index.js

```diff
@@ -18,6 +18,19 @@ const ec2Client = new EC2({customUserAgent});
 const dbClient = AWSXRay.captureAWSv3Client(new DynamoDB({customUserAgent}));
 const docClient = DynamoDBDocument.from(dbClient);
 
+const ORG_AGGREGATOR_ERROR = 'The configuration aggregator cannot be created because your aggregator source type changed for that aggregator.'
+const DUPLICATE_ACCOUNTS_ERROR =
+    'Your configuration aggregator contains duplicate accounts. Delete the duplicate accounts and try again.';
+
+function handleAwsConfigErrors(err) {
+    // when using AWS Organizations, we do not want to update the the aggregator
+    if ([DUPLICATE_ACCOUNTS_ERROR, ORG_AGGREGATOR_ERROR].includes(err.message)) {
+        console.log(err);
```

Comment:
should we not use error or warning for the log type?

Reply:
I'm going to change this Lambda function to use a proper logging library so we can fix that then.